Gaikan/UIKit/Renderer/ViewStyleRenderer.swift

```diff
@@ -12,12 +12,17 @@ internal class ViewStyleRenderer {
     final class func render(stylable: UIView, styleRule: StyleRule) {
         let border = styleRule.border ?? Border(width: 0, color: nil)
         let visible = styleRule.visible ?? true
+        let corners = styleRule.corners ?? Corners(radius: 0)
+        
+        stylable.layer.cornerRadius = corners.radius
+        stylable.clipsToBounds = true
+        stylable.layer.masksToBounds = true
```

Comment:
I think we should let the user opt-in for these options. This might have performance issues and conflicts with some other properties, like shadows.